music/demos/ddsp.ts

```diff
@@ -0,0 +1,176 @@
+/**
+ * @license
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { MODEL } from '../src/ddsp/constants';
+import * as mm from '../src/index';
+
+function floatTo16BitPCM(output, offset, input) {
```

Comment:
It seems like some of these functions would go good in a general "audio utils" library. What do you think @notwaldorf @adarob ?

Reply:
this is a function used in the demo actually, I don't think the library should try to handle this but I'll leave it up to you guys to decide.